app/src/components/pages/divisionsPage/divisionsPage.tsx

```diff
@@ -46,9 +46,9 @@ export default function DivisionsPage({ data }) {
             />
           ) : (
             <>
-              <CollectionLanesLoading />,
-              <CollectionLanesLoading />,
-              <CollectionLanesLoading />
+              {[...Array(36)].map((_, index) => (
```

Comment:
Where did you get "36" from?

Reply:
Randomish: needed to be more than 24, the number of divisions, but also needed to somewhat account for the space taken up by loading the hero and the footer

Reply:
yeah good question - looking at the sheet lana provided https://docs.google.com/spreadsheets/d/1oDILU2vJBKqTcv0nINuDkfwmp9hIyZc_0jzHcadSGrY/edit?gid=0#gid=0 there are less than 36 divisions to display

Reply:
I think you can get away with a much smaller number to display the mocked skeleton loaders. Perhaps just 6 or so? Not sure the user will scroll down quickly enough to see the rest of the loaders.

Reply:
If the user is anywhere near the footer and reloads, I assumed we want to return them to where they were